components/dom_crawler.rst

```diff
@@ -231,6 +231,7 @@ and :phpclass:`DOMNode` objects:
         $html = $crawler->html();
 
     The ``html`` method is new in Symfony 2.3.
+    In PHP < 5.3.6 it might return html entities which are not properly decoded.
```

Comment:
I think we should put this in a `caution` directive (and we should reword this a bit):

```rst
.. caution::

    Due to an issue in PHP, the ``html()`` method returns wrongly decoded HTML
    entities in PHP versions lower than 5.3.6 (for example, it returns ``•`` instead
    of ``&bull;``).
```

@jakzal Do you know if that issue affects all HTML entities or only a subset?